test/Platform.spec.ts

```diff
@@ -51,8 +52,39 @@ describe('Platform', () => {
             expect(playerMock.y).to.be.lessThan(playerGroundedPos);
         });
 
-        it('should not allow player to jump if collided with from any other side', () => {
-            assert.fail('Not implemented');
+        it('should not allow player to jump if collided with from any other side', async () => {
+            // Setup Blocks
+            block = scene.createBlock('reg', 400, 400);
+            scene.createBlock('reg', 200, 200);
+            // Constants
+            const playerGroundedPos =
+                block.getBounds().top - playerMock.getBounds().height / 2;
+            // Position player
+            playerMock.x = 400;
+            playerMock.y = 200;
+            // Wait for jump
+            await phaserTester.waitForGameEvent('jump');
```

Comment:
How about have the player be in jump state at the start of the test so that:
- it's less confusing to understand
- less complex

Reply:
To continue work in #52.